### PR TITLE
feat([sc-8464]): ext nft transfer

### PR DESCRIPTION
**Pull request by chorniy8pavlo**

### CODE REVIEW

File: packages/embed/src/lib/nfid.ts [hunk 1]
```diff
@@ -222,6 +222,33 @@ export class NFID {
...
+  }: {
+    receiver: string;
+    tokenId: string;
+    sourceAddress: string;
+  }) {
+    console.log('NFID.requestTransferNFT');
+    if (!NFID.nfidIframe) throw new Error('NFID iframe not instantiated');
+    showIframe();
+    const iframe = getIframe();
+    const response = await request(iframe, {
```
Comment (plitzenberger) [inline packages/embed/src/lib/nfid.ts:238]: Please type the response here:
https://github.com/internet-identity-labs/sdk-ts/blob/8456babf79b2145014870f3b0f84bc12fbb74c6d/packages/embed/src/lib/postmsg-rpc.ts#L53-L63

Comment (plitzenberger) [reply packages/embed/src/lib/nfid.ts:238]: And, would be nice to stick to the interface standard:
https://github.com/internet-identity-labs/sdk-ts/blob/8456babf79b2145014870f3b0f84bc12fbb74c6d/packages/embed/src/lib/postmsg-rpc.ts#L17-L19

**Review by plitzenberger — CHANGES_REQUESTED** [review]
types 🙏

File: packages/embed/src/lib/nfid.ts (hunk 1 above)
Comment (chorniy8pavlo) [reply packages/embed/src/lib/nfid.ts:238]: Fungible and non-fungible have the same response interface

Comment (plitzenberger) [reply packages/embed/src/lib/nfid.ts:238]: got it. But the return type is still wrong. Should be `result`

**Review by plitzenberger — APPROVED** [review]
lgtm

File: packages/embed/src/lib/postmsg-rpc.ts [hunk 2]
```diff
@@ -36,7 +36,9 @@ type RPCRequestMetadata = {
   rpcUrl?: string;
 };
 
-type Method = 'ic_getDelegation' /* add more method names as needed */;
+type Method =
+  | 'ic_getDelegation'
+  | 'ic_requestTransfer' /* add more method names as needed */;
```
Comment (plitzenberger) [inline packages/embed/src/lib/postmsg-rpc.ts:42]: @chorniy8pavlo this is also required to map to the right response type

**Review by ghost — APPROVED** [review]

**Pull request merged**